Dockerfile

```diff
@@ -4,8 +4,9 @@ WORKDIR /code
 COPY ./static/package*.json /code/static/
 RUN cd /code/static && npm install
 
-# Main image
-FROM python:3.7
+# Base build
+FROM python:3.7 as base
+RUN addgroup -gid 10001 --system simplelogin && adduser --uid 10000 --system --ingroup simplelogin --home /home/simplelogin simplelogin
```

Comment:
I'd suggest moving this user creation to a future PR to avoid any issue of reading files/logs from the Docker container and to have this PR focusing on having a complete working setup with docker-compose.